### PR TITLE
Api create report

### PR DESCRIPTION
**Pull request by quocquann**
## Related Tickets
[- ticket redmine](https://edu-redmine.sun-asterisk.vn/issues/70943)

## HOW
Tạo report

## Evidence (Screenshot or Video)
image

### CODE REVIEW

**Comment by quocquann** [comment]
ready

File: api/serializers.py [hunk 1]
```diff
@@ -342,4 +342,10 @@ def validate(self, data):
...
     class Meta:
         model = Task
-        fields = ('content', 'start_date', 'end_date', 'status', 'user')
+        fields = ("content", "start_date", "end_date", "status", "user")
+
+
+class ReportSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Report
+        fields = ["content"]
```
Comment (huongnt-2545) [inline api/serializers.py:351]: thêm validate require cho field content nhé

Comment (quocquann) [reply api/serializers.py:351]: image
e dùng ModelSerializer nên nó check bên model r ạ

**Comment by quocquann** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]

**Pull request merged**